CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Implemented conditional functions - ``coalesce()``.
```

Comment:
as discussed, pls remove now and add this later on the story branch after all implementations are done